v0/destinations/googlesheets/transform.js

```diff
@@ -87,61 +88,68 @@ const processWithCustomMapping = (message, attributeKeyMapping) => {
   return responseMessage;
 };
 
+const batch = events => {
+  return { batch: events };
+};
+
 // Main process Function to handle transformation
 const process = event => {
   const { message, destination } = event;
   if (destination.Config.sheetName) {
     const payload = {
-      message: processWithCustomMapping(
-        message,
-        destination.Config.eventKeyMap
-      ),
-      spreadSheetId: destination.Config.sheetId,
-      spreadSheet: destination.Config.sheetName
+      message: processWithCustomMapping(message, destination.Config.eventKeyMap)
```

Comment:
Rather than having this breaking change, let it still return the other fields. Just skip them at the receiving end.

Reply:
Added backward compatibility support for Processor Transform